jinja2/base.html

```diff
@@ -176,5 +180,6 @@
   {% endblock %}
 
   {% block js %}{% endblock %}
+  <script src="https://cdn.speedcurve.com/js/lux.js?id=108906238" async defer crossorigin="anonymous"></script>
```

Comment:
Given that this script is async defer, can we just add it to the end of speedcurve-lux.html for simplicity? That would presumably mean a performance hit for browsers that don't support those things, but I'm not sure if we care?

Reply:
caniuse.com says that async and defer are both supported by IE10 and IE11...